client/src/js/base/SelectBox.js

```diff
@@ -16,3 +18,12 @@ export const SelectBox = styled(Box)`
         padding-top: 5px;
     }
 `;
+
+export const SelectBox = props => {
+    const buttonType = props.type ? props.type : "button";
+    return (
+        <StyledSelectBox {...props} type={buttonType} as="button">
+            {props.children}
+        </StyledSelectBox>
+    );
+};
```

Comment:
Why can't `button` type just be hardcoded?

Reply:
In retrospect, I think it can be in this case.  Seems unlikely there would be a use case where we want to use a select box to submit. Committed a new version that removes the helper function and has a hardcoded type.